package.json

```diff
@@ -388,6 +388,7 @@
 		"@types/request-promise": "^4.1.48",
 		"@types/shelljs": "^0.8.10",
 		"@types/tmp": "^0.2.3",
+		"ansi-regex": "^5.0.1",
```

Comment:
shouldn't it be added to `devDependencies` instead of `dependencies`?

Reply:
also is it normal that the package-lock.json was not updated given that a dependency has been added?

Reply:
> shouldn't it be added to `devDependencies` instead of `dependencies`?

I will try to replace

> also is it normal that the package-lock.json was not updated given that a dependency has been added?

Interesting question it was needed to add it into the `package` but no error according to `package-lock`

Reply:
the clean way could be add this dependency through `npm install --save-dev ansi-regex@5.0.1` which should update both - package.json and package-lock.json

Reply:
Package moved to `devDependencies`, I have also generated a new `package-lock`, but after a short manual look I decided not to change it due to the uselessness of the changes (some empty brackets and elements swapping)